### ISSUE TITLE
AnalyticalTable: screenreader does not say the role of selection checkboxes

### ISSUE THREAD

**Issue by cmealares** [UI5/webcomponents-react#8697]
### Describe the bug

When the first column of the AnalyticalTable contains checkboxes (for selection), the screenreader
(jaws) does not correctly identify the checkboxes. It does not inform about their role.

### Isolated Example

WC4R's storybook

### Reproduction steps

1. Open WC4R storybook and go to the Analytical example
2. Click on "default"
3. Set this option: **SelectionMode=Multiple** (checkboxes are now displayed)
4. Set focus on one of the checkboxes

Here is what Jaws says:

> To select all rows, press the spacebar. Selection Column
> To select the row, press the spacebar.
> Column 1

### Expected Behaviour

The role of UI elements must be announced to the user.
This is a requirement of SAP's product standard ACC-264.1
See https://wiki.one.int.sap/wiki/spaces/Acc/pages/2070649167/ACC-264+Details

### UI5 Web Components for React Version

V1 and V2

### UI5 Web Components Version

V1 and V2

### Browser

Chrome

### Relevant log output

```shell

```

### Organization

SAP

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @cmealares 

the selection checkboxes are only visual indicators and cannot be focused by keyboard navigation and should therefore not be picked up by screen-readers. This is achieved using the `aria-hidden` attribute. So, this behavior is by design.

I'm not sure in which version you found the checkboxes to be focusable, but in (the latest) v2, it shouldn't be possible.
